BlueprintUI/Sources/Layout/Hidden.swift

```diff
@@ -0,0 +1,45 @@
+import CoreGraphics
+
+/// `Hidden` conditionally hides its wrapped element.
+///
+/// - Note: When an element is hidden, any elements within the wrapped element will be hidden.
+public struct Hidden: Element {
+    public var isHidden: Bool
+    public var wrappedElement: Element
+
+    public init(_ isHidden: Bool = true, wrapping element: Element) {
+        self.isHidden = isHidden
+        wrappedElement = element
+    }
+
+    public var content: ElementContent {
+        ElementContent(child: wrappedElement, layout: Layout(isHidden: isHidden))
+    }
+
+    public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
+        nil
+    }
+
+    private struct Layout: SingleChildLayout {
+        var isHidden: Bool
+
+        func measure(in constraint: SizeConstraint, child: Measurable) -> CGSize {
+            child.measure(in: constraint)
+        }
+
+        func layout(size: CGSize, child: Measurable) -> LayoutAttributes {
+            var attributes = LayoutAttributes(size: size)
```

Comment:
Outloud thought: It'd be nice if `LayoutAttributes` had a configure block to clean up these callsites:

```
LayoutAttributes(size: size) {
   $0.isHidden = isHidden
}
```